packages/flutter/lib/src/widgets/fade_in_image.dart

```diff
@@ -390,6 +438,7 @@ class _FadeInImageState extends State<FadeInImage> {
       width: widget.width,
       height: widget.height,
       fit: fit,
+      filterQuality: filterQuality ?? FilterQuality.low,
```

Comment:
make `FilterQuality.low` the default value of the `filterQuality` param on the widget and make that non-nullable?

Reply:
should be done now